lib/ssl/src/ssl_certificate.erl

```diff
@@ -509,7 +524,8 @@ maybe_shorten_path(Path, PartialChainHandler, Default) ->
     %% public_key:pkix_path_validation by letting
     %% the user put its trust in an intermidate cert
     %% from the certifcate chain sent by the peer.
-    try PartialChainHandler(Path) of
+    DerCerts = [Der || {Der, _} <- Path],
+    try PartialChainHandler(DerCerts) of
```

Comment:
Why can't PartialChainHandler take the Path as is? Is that because it is in the public API?
Does this mean that the `PartialChainHandler` need to decode the certificates although the are already decoded?
I suppose the PartialChainHandler is provided by the user?

Reply:
That would be an API change.

Reply:
Yes